### ISSUE TITLE
bị hỏng

### ISSUE THREAD

**Issue by zedmaster9** [codesandbox/codesandbox-client#4223]
<h1>💥 Crash Report</h1>

  <h2>What were you trying to accomplish when the crash occurred?</h2>

  > Please use this issue template to describe what you were doing when you encountered this crash. While we are able to fill in some details automatically, it's not always enough to reproduce!

  <h3>Link to sandbox: [link]() (optional)</h3>

  <h3>Crash Details</h3>

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System | CodeSandbox Version |
  | ------- | --------- | ---------------- | ------------------- |
  | chrome | 80.0.3987 | Windows 10           | a72a08c2e          |

  **Route:**
  https://codesandbox.io/s/m7q0r29nn9

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
  Error: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
    at HTMLDivElement.removeChild (<anonymous>)
    at ys (https://codesandbox.io/static/js/common.d8af35bee.chunk.js?_sw-precache=19b1603c7e88af76e6ba0a3eadc9a1bd:1:1758329)
    at ms (https://codesandbox.io/static/js/common.d8af35bee.chunk.js?_sw-precache=19b1603c7e88af76e6ba0a3eadc9a1bd:1:1756330)
    at ys (https://codesandbox.io/static/js/common.d8af35bee.chunk.js?_sw-precache=19b1603c7e88af76e6ba0a3eadc9a1bd:1:1758066)
    at yl (https://codesandbox.io/static/js/common.d8af35bee.chunk.js?_sw-precache=19b1603c7e88af76e6ba0a3eadc9a1bd:1:1777331)
    at https://codesandbox.io/static/js/common.d8af35bee.chunk.js?_sw-precache=19b1603c7e88af76e6ba0a3eadc9a1bd:1:1798863
    at qo (https://codesandbox.io/static/js/common.d8af35bee.chunk.js?_sw-precache=19b1603c7e88af76e6ba0a3eadc9a1bd:1:1723236)
    at bl (https://codesandbox.io/static/js/common.d8af35bee.chunk.js?_sw-precache=19b1603c7e88af76e6ba0a3eadc9a1bd:1:1775216)
    at il (https://codesandbox.io/static/js/common.d8af35bee.chunk.js?_sw-precache=19b1603c7e88af76e6ba0a3eadc9a1bd:1:1765543)
    at https://codesandbox.io/static/js/common.d8af35bee.chunk.js?_sw-precache=19b1603c7e88af76e6ba0a3eadc9a1bd:1:1723527
    at https://codesandbox.io/static/js/common.d8af35bee.chunk.js?_sw-precache=19b1603c7e88af76e6ba0a3eadc9a1bd:1:1798863
    at qo (https://codesandbox.io/static/js/common.d8af35bee.chunk.js?_sw-precache=19b1603c7e88af76e6ba0a3eadc9a1bd:1:1723236)
    at Yo (https://codesandbox.io/static/js/common.d8af35bee.chunk.js?_sw-precache=19b1603c7e88af76e6ba0a3eadc9a1bd:1:1723472)
    at $o (https://codesandbox.io/static/js/common.d8af35bee.chunk.js?_sw-precache=19b1603c7e88af76e6ba0a3eadc9a1bd:1:1723407)
    at se (https://codesandbox.io/static/js/common.d8af35bee.chunk.js?_sw-precache=19b1603c7e88af76e6ba0a3eadc9a1bd:1:1792615)
    at Sn (https://codesandbox.io/static/js/common.d8af35bee.chunk.js?_sw-precache=19b1603c7e88af76e6ba0a3eadc9a1bd:1:1707033)
  
    in Files___StyledSidebarRow
    in div
    in t
    in Styled(styled.div)
    in section
    in t
    in Styled(styled.div)
    in Unknown
    in t
    in Files___StyledCollapsible
    in Ol
    in Cc
    in project-summary
    in div
    in t
    in Unknown
    in div
    in t
    in elements__Container
    in Unknown
    in WithTheme(Component)
    in div
    in t
    in div
    in t
    in div
    in div
    in t
    in styled.div
    in div
    in t
    in elements__Container
    in t
    in Unknown
    in Unknown
    in Unknown
    in t
    in t
    in div
    in t
    in elements__Content
    in a
    in withRouter(a)
    in div
    in t
    in elements__Container
    in Unknown
    in withRouter()
    in t
    in Unknown
    in t
    in c
    in t
  ```

  </details>

**Comment by garethx** [comment]
@zedmaster9 What were you trying to accomplish when the crash occurred?

**Comment by SaraVieira** [comment]
closed by #4571
